numpy/random/setup.py

```diff
@@ -126,6 +126,15 @@ def generate_libraries(ext, build_dir):
                          depends=['mtrand.pyx'],
                          define_macros=defs + LEGACY_DEFS,
                          )
+    config.add_data_files('bit_generator.pxd')
+    config.add_data_files('bounded_integers.pxd')
+    config.add_data_files('common.pxd')
+    # config.add_data_files('generator.pxd')
+    # config.add_data_files('mt19937.pxd')
+    # config.add_data_files('mtrand.pxd')
+    # config.add_data_files('pcg64.pxd')
+    # config.add_data_files('philox.pxd')
+    # config.add_data_files('sfc64.pxd')
```

Comment:
Still need to create these pxd files so users can do `from pcg64 cimport PCG64` to use the BitGenerator directly from cython

Reply:
Why not keep this in the main namespace? `PCG64` is in `numpy.random` so I'd much prefer it not be in separate `pxd` files here

Reply:
> main namespace

This mimics the way we import the BitGenerators now.
Do you mean we should refactor the module so that [`__init__.py`](https://github.com/numpy/numpy/blob/v1.17.2/numpy/random/__init__.py#L186) imports differently?

Reply:
That's already done remember? gh-14360

Reply:
I am still seeing the BitGenerators in both the c-extension module and as imported by `__init__.py`:
```
>>> np.random.mt19937.MT19937
<class 'numpy.random.mt19937.MT19937'>
>>> np.random.MT19937
<class 'numpy.random.mt19937.MT19937'>
>>> 
```
I am not sure we can flatten the namespace without unifying all the c-extension modules into one

Reply:
Hmm, missing some more underscores.

> I am not sure we can flatten the namespace without unifying all the c-extension modules into one

Of course we can, it's just a matter of adding underscores to the `.pyx` files.

Reply:
made private in 9bdf1749e. `mtrand` must remain public as per NEP 19 

> ... the global `RandomState` instance MUST be accessible in this initial release by the name `numpy.random.mtrand._rand` ...